src/ui/mainwindow.cpp

```diff
@@ -184,6 +193,8 @@ MainWindow::change_directory(std::string dir) {
 
 void
 MainWindow::play_pause_button_clicked() {
+    Logger::init_logging()->log(LogLevel::INFO, LogOutput::CONSOLE, "Play/Pause button clicked");

```

Comment:
Let's not log this. Again, not useful information and will only spam the log. Imagine opening the log file and see
```
[INFO] Play/Pause button clicked
[INFO] Play/Pause button clicked
[INFO] Play/Pause button clicked
[INFO] Play/Pause button clicked
[INFO] Play/Pause button clicked
[INFO] Play/Pause button clicked
[INFO] Play/Pause button clicked
[INFO] Play/Pause button clicked
[INFO] Play/Pause button clicked
[INFO] Play/Pause button clicked
[INFO] Play/Pause button clicked
[INFO] Play/Pause button clicked
[INFO] Play/Pause button clicked
[INFO] Play/Pause button clicked
[INFO] Play/Pause button clicked
```
Pretty annoying right?